### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by ghtina**
in coop with #42 done
need review

### CODE REVIEW

**Comment by ocastx** [comment]
- [x] Werden korrekte Typen benutzt?
- [x] Methoden haben korrekte Sichtbarkeit
- [x] Variablen haben korrekte Sichtbarkeit
- [ ] Keine Rechtschreibfehler (Schrift/Grammatik)
- [x] Code gut verständlich
- [x] Eindeutige Variablennamen
- [x] Dokumentation in Form von JavaDoc vorhanden
- [x] Keine logischen Fehler
- [ ] Checkstyle keine (drastischen) Fehler
- [x] FindBugs keine (drastischen) Fehler

File: plugins/de/opendiabetes/vault/plugin/importer/mysugr/package-info.java [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /**
- * This package holds this MySugrCsvImporter plugin.
+ * This package holds the MySugrCsvImporter plugin
+ * and its specifically needed classes..
```
Comment (ocastx) [inline plugins/de/opendiabetes/vault/plugin/importer/mysugr/package-info.java]: One dot to much ;)

File: src/de/opendiabetes/vault/data/VaultDao.java [hunk 2]
```diff
@@ -117,6 +117,9 @@ public static void finalizeDb() throws IOException {
      */
     public static void initializeDb() throws SQLException {
         //TODO combine logging
+        // Der OR Mapper benutzt (soweit ich mich erinnere) Log4J zum loggen und ich möchte gerne den Logger
```
Comment (ocastx) [inline src/de/opendiabetes/vault/data/VaultDao.java]: Either provide comment in english, fix or leave it out completely and specify as task

**Review by ocastx — CHANGES_REQUESTED** [review]
Only small changes needed, after that it can be merged

File: plugins/de/opendiabetes/vault/plugin/exporter/sourcecode/SourceCodeExporter.java [hunk 3]
```diff
@@ -64,6 +64,11 @@ public SourceCodeExporter(final PluginWrapper wrapper) {
     @Extension
     public static class SourceCodeExporterImplementation extends VaultExporter {
 
+        /**
+         * List to hold all the entries queried from the database in {@link #prepareData(List)}
```
Comment (ocastx) [inline plugins/de/opendiabetes/vault/plugin/exporter/sourcecode/SourceCodeExporter.java:68]: CheckStyle wants a dot at the end of the first line

File: src/de/opendiabetes/vault/container/VaultEntryAnnotation.java [hunk 4]
```diff
@@ -97,15 +97,19 @@ public VaultEntryAnnotation setValue(final String value) {
...
     /**
      * Converting VaultEntries to string.
+     *
      * @return the VaultEntry as string.
-     * TODO.
      */
     @Override
     public String toString() {
-        return type.toString();
+        return "VaultEntryAnnotation{" +
```
Comment (ocastx) [inline src/de/opendiabetes/vault/container/VaultEntryAnnotation.java]: CheckStyle: "+" needs to be in the new line

File: src/de/opendiabetes/vault/container/VaultEntryAnnotation.java [hunk 5]
```diff
@@ -133,7 +137,7 @@ public String toString() {
      * @return The VaultEntry as string.
      */
     public String toStringWithValue() {
-        return value.isEmpty() ? this.toString() : this.toString() + "=" + value;
+        return value.isEmpty() ? this.getType().toString() : this.getType().toString() + "=" + value;
```
Comment (ocastx) [inline src/de/opendiabetes/vault/container/VaultEntryAnnotation.java]: CheckStyle: No Inline-Conditional Operator

File: src/de/opendiabetes/vault/data/VaultDao.java [hunk 6]
```diff
@@ -404,55 +407,56 @@ public VaultEntry queryVaultEntryById(final long id) {
...
-     */
-    public boolean removeEntry(final VaultEntry historyEntry) {
-        try {
-            vaultDao.deleteById(historyEntry.getId());
-            LOG.log(Level.INFO, "Removed dntry: {0}", historyEntry.toString());
-        } catch (SQLException exception) {
-            LOG.log(Level.SEVERE, "Error while db query", exception);
-            return false;
-        }
+
```
Comment (ocastx) [inline src/de/opendiabetes/vault/data/VaultDao.java]: CheckStyle: Trailing Spaces

**Review by ocastx — CHANGES_REQUESTED** [review]

**Comment by lucasbuschlinger** [comment]
@ocastx Did requested changes and also other things mentioned by Checkstyle, such as removing all inline conditionals (I find it generally more readable with if/else)

Please re-review :)

**Review by ocastx — APPROVED** [review]

**Pull request merged**